src/assemblyItemsBrowser.js

```diff
@@ -108,11 +109,17 @@ module.exports = React.createClass({
 		return ret
 	},
 
-	resolveAddress: function(address)
+	resolveAddress: function(address, props)
```

Comment:
I think the name does not fit very well, this function is about loading code, not about resolving addresses.